django/src/rdwatch_scoring/views/vector_tile.py

```diff
@@ -0,0 +1,234 @@
+
+    # Generate the vector tiles and cache them if there's no hit
+    if tile is None:
+        envelope = Func(z, x, y, function='ST_TileEnvelope')
+        intersects = Q(
+            Func(
+                'transformedgeom',
+                envelope,
+                function='ST_Intersects',
+                output_field=BooleanField(),
+            )
+        )
+        transform = Func(
+            'geomfromtext', 3857, function='ST_Transform', output_field=GeometryField()
+        )
+        mvtgeom = Func(
+            'transformedgeom',
+            envelope,
+            function='ST_AsMVTGeom',
+            output_field=Field(),
+        )
+        geomfromuniongeometrytext = Func(
+            'union_geometry', 4326, function='ST_GeomFromText', output_field=Field()
+        )
+        geomfromobservationtext = Func(
+            'geometry', 4326, function='ST_GeomFromText', output_field=Field()
+        )
+
+        site_queryset = (
+            Site.objects.filter(evaluation_run_uuid=evaluation_run_uuid)
+            .alias(geomfromtext=geomfromuniongeometrytext)
+            .alias(transformedgeom=transform)
+            .filter(intersects)
+            .values()
+            .annotate(
+                id=F('site_id'),
+                mvtgeom=mvtgeom,
+                configuration_id=F('evaluation_run_uuid'),
+                label=Case(
+                    When(
+                        Q(status_annotated__isnull=False),
+                        Lower(Replace('status_annotated', Value(' '), Value('_'))),
+                    ),
+                    default=Value('unknown'),
+                ),  # This needs a version to be scoring coloring,
+                # but that needs some coordination with kitware
+                timestamp=ExtractEpoch('evaluation_run_uuid__start_datetime'),
+                timemin=ExtractEpoch('start_date'),
+                timemax=ExtractEpoch('end_date'),
+                performer_id=F('originator'),
+                region=F('region_id'),
+                groundtruth=Case(
+                    When(
+                        Q(originator='te'),
+                        True,
+                    ),
+                    default=False,
+                ),
+                site_polygon=Value(False, output_field=BooleanField()),
+            )
+        )
+        (
+            site_sql,
+            site_params,
+        ) = site_queryset.query.sql_with_params()
+
+        observations_queryset = (
+            Observation.objects.filter(
+                site_uuid__evaluation_run_uuid=evaluation_run_uuid
+            )
+            .alias(geomfromtext=geomfromobservationtext)
+            .alias(transformedgeom=transform)
+            .filter(intersects)
+            .values()
+            .annotate(
+                id=F('uuid'),
+                mvtgeom=mvtgeom,
+                configuration_id=F('site_uuid__evaluation_run_uuid'),
+                site_number=F('site_uuid__site_id'),
```

Comment:
```suggestion
                site_number=Substr(F('site_uuid__site_id'), 9),  # pos is 1 indexed
```
The site_number was only meant to be the number instead it is the whole region+number+performer I.E: 'KR_R001_0012_te_2.0.1'
Using the [Substr](https://docs.djangoproject.com/en/4.2/ref/models/database-functions/#substr) function and starting at position 9 should work.
![image](https://github.com/ResonantGeoData/RD-WATCH/assets/61746913/fad05c6e-0d06-4054-abef-971be1b6858c)
![image](https://github.com/ResonantGeoData/RD-WATCH/assets/61746913/6a670e3c-88c2-49d7-933b-fb9076c4a0c3)

This will also require importing Substr `from django.db.models.functions`